product_tag/__manifest__.py

```diff
@@ -4,29 +4,21 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Product Tag",
-    "summary": "Tag your product",
-    "version": "8.0.1.0.0",
-    "category": "E-commerce",
-    "website": "www.akretion.com",
-    "author": " Akretion",
-    "license": "AGPL-3",
-    "application": False,
-    'installable': False,
-    "external_dependencies": {
-        "python": [],
-        "bin": [],
-    },
-    "depends": [
-        "product",
-    ],
-    "data": [
-        "views/product_tag_view.xml",
-        "views/product_view.xml",
-        "security/ir.model.access.csv",
+    'name': 'Product Tag',
+    'version': '10.0.1.0.0',
+    'author': "Akretion, "
+              "Serpent Consulting Services Pvt. Ltd.",
+    'website': 'http://www.akretion.com',
+    'license': 'AGPL-3',
+    'category': 'E-commerce',
+    'summary': 'Tag your product',
+    'depends': [
+        'product', 'sales_team'
     ],
-    "demo": [
+    'data': [
+        'views/product_tag_view.xml',
+        'views/product_view.xml',
+        'security/ir.model.access.csv',
     ],
-    "qweb": [
-    ]
+    'installable': False,
```

Comment:
make it true now

Reply:
@JayVora-SerpentCS ,
I have noticed that by default the `installable` value is `False` before migration.
See [Reference](https://github.com/SerpentConsultingServices/odoo-shopinvader/blob/10.0-to-migrate/product_tag/__manifest__.py#L15)
But I will make it `True`.